src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -119,23 +150,11 @@ static void updateAttributeLinks(EndpointId endpoint)
             // which would be the EffectiveControlMode is dependant on the actual
             // physical pump application running "on-top" of this cluster server.
             Attributes::EffectiveControlMode::Set(endpoint, controlMode);
-            pumpStatus.Clear(PumpStatus::kRemoteFlow);
-            pumpStatus.Clear(PumpStatus::kRemotePressure);
-            pumpStatus.Clear(PumpStatus::kRemoteTemperature);
-            switch (controlMode)
+            if (isPumpStatusAvailable)
             {
-            case PumpControlMode::kConstantFlow:
-                pumpStatus.Set(PumpStatus::kRemoteFlow);
```

Comment:
What happened to these `Set` bits?

Reply:
Well, my take is that these should not be there since the bits relates to the use of remote sensors and this is the case where no sensors are available. They should be cleared and none of them should be set.

Reply:
OK.